rpm/droidmedia.spec

```diff
@@ -8,7 +8,7 @@
 
 Name:          droidmedia
 Summary:       Android media wrapper library
-Version:       0.0.0
+Version:       0.20170214.0
```

Comment:
Let's take this in the too, it'll be useful until https://github.com/mer-hybris/droid-hal-device/pull/159 gets merged